commcare_connect/utils/ocs_api.py

```
@@ -0,0 +1,75 @@
+    }
+    payload.update({k: v for k, v in optionals.items() if v is not None})
+
+    try:
+        response = httpx.post(
+            f"{settings.OCS_BASE_URL}/api/trigger_bot",
+            json=payload,
+            headers={"Authorization": f"Bearer {token.token}"},
+        )
+        response.raise_for_status()
+    except httpx.HTTPStatusError:
+        raise OcsApiError(f"Failed to trigger bot: {response.text}")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)